package.json

```diff
@@ -28,6 +28,7 @@
     "node-os-utils": "^1.2.2"
   },
   "dependencies": {
+    "body-parser": "^1.19.0",
```

Comment:
No need to install body parser. You can access most of the things you would use body-parser for as methods on the main express object.
```suggestion
```